src/App.tsx

```diff
@@ -10,32 +11,22 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0]);
```

Comment:
i know this and i tried to do this task using the id first. I realized it would be much more logical to store the whole tab there, not only the id as I need access to the title in the app component

implementing this via tab.id in the state would lead to more troubles rather than to the logical solution

Reply:
then just rename state name